examples/SwissLipids/11.ttl

```diff
@@ -5,11 +5,11 @@
 
 ex:11 a sh:SPARQLExecutable,
         sh:SPARQLSelectExecutable ;
-    rdfs:label "Mapping HMDB to SwissLipids" ;
+    rdfs:label "Mapping HMDB to SwissLipids"@en ;
```

Comment:
why do we have rdfs:label instead of schema:keywords? Do we use both, for different purposes? We should also document this.